modules/dxp/apps/multi-factor-authentication/app.bnd

```diff
@@ -0,0 +1,15 @@
+Liferay-Releng-App-Description:
+Liferay-Releng-App-Title: ${liferay.releng.app.title.prefix} Multi-Factor Authentication
+Liferay-Releng-Bundle: true
+Liferay-Releng-Category: Security
+Liferay-Releng-Demo-Url:
+Liferay-Releng-Deprecated: false
+Liferay-Releng-Fix-Delivery-Method: core
+Liferay-Releng-Labs: false
+Liferay-Releng-Marketplace: true
+Liferay-Releng-Portal-Required: false
+Liferay-Releng-Public: ${liferay.releng.public}
+Liferay-Releng-Restart-Required: false
```

Comment:
@gaborlovasdotcom do we have the poshi test to be able to set this to false? or should we mark it as true in the meantime?

Reply:
it is listed here https://grow.liferay.com/people/App.bnd as a requirement

Reply:
@csierra looks like no:
```
(master) $ git grep "hot.deploy.osgi.app.includes"
test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/CrossClusterReplicationES7.testcase:              property hot.deploy.osgi.app.includes = "portal-search-elasticsearch-cross-cluster-replication";
test/functional/com/liferay/portalweb/tests/coreinfrastructureee/search/elasticsearch/Elasticsearch7EE.testcase:                property hot.deploy.osgi.app.includes = "portal-search-learning-to-rank";
test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:        
        property hot.deploy.osgi.app.includes = "saml";
```
And you can always change it to false later if Gabor doesn't have time to automate right now :).

Reply:
I have some question about this what I shared with you @csierra .

Reply:
As discussed with @xbrianlee, the test is not needed.